src/main/java/org/opentripplanner/transit/raptor/api/request/DebugRequestBuilder.java

```diff
@@ -56,7 +57,7 @@ public DebugRequestBuilder addStops(int ... stops) {
     }
 
     public DebugRequestBuilder addPath(Collection<Integer> path) {
-        if(!path.isEmpty()) {
+        if(!this.path.isEmpty()) {
```

Comment:
`this.path` could be renamed to `paths` to avoid confusion.

Reply:
I guess both `this.path` and `path`(the argument) should be renamed. Will do.

Reply:
Sorry, it is not paths in plural - it is a single path represented by the stops it visit.